cmd/cli/user.go

```diff
@@ -24,13 +30,16 @@ func transferUserSubscriptionsAndContacts(database moira.Database, from, to stri
 	if err != nil {
 		return err
 	}
+
 	subscriptions, err := database.GetSubscriptions(subscriptionIDs)
 	if err != nil {
 		return err
 	}
+
 	for _, subscription := range subscriptions {
 		subscription.User = to
```

Comment:
nil pointer again

Reply:
I am not sure if this is a problem. Offering to exit the program? Since it will not cause a particular problem, I decided to leave it, except for a cherished processor time.
https://play.golang.org/p/5chouKdtBTU

Reply:
what type of data does GetSubscriptions return?
why do you check it's length now?